students/grzegorz_bajorski/lesson_10_organizing_files/filling_in_the_gaps.py

```diff
@@ -0,0 +1,24 @@
+import os
+import fnmatch
+import shutil
+
+
+prefix = "spam*.txt"
+files_list = []
+
+for filename in os.listdir():
+    if fnmatch.fnmatch(filename, prefix):
+        files_list.append(filename)
+
+files_list.sort()
+
+prev_num = 0
+for files in files_list:
+    current_num = int(files.split("spam")[1].split(".")[0])
+    expected_num = prev_num + 1
+    if current_num != expected_num:
+        new_name = "spam" + str(expected_num) + ".txt"
```

Comment:
To keep proper file name conventions (spam00X) you should pad string on the left with zeros to fill width using ```zfill(3)```